canonical-web-and-design/juju.is
# Build the about operator-services page

## QA
- Go to http://0.0.0.0:8041/about/operator-services
- Check the content matches [the copy doc](https://docs.google.com/document/d/1_HS6kPV3jxSA0k9d-nr5PFu9V40Nv7d00N5tBwcPmG8/edit#)

Fixes https://github.com/canonical-web-and-design/juju.is/issues/103
